mirage/factories/coupon.js

```diff
@@ -0,0 +1,4 @@
+import { Factory } from 'ember-cli-mirage';
+
+export default Factory.extend({
+});
```

Comment:
Would it make sense to define coupons' attributes in factory, so that it could be generated automatically? E.g.
```js
  name() { 
    return faker.lorem.words(1);
  },
  percentageOff() { 
    return faker.random.number({min:5, max:10});
  },
  amountOff() { 
    return faker.random.number({min:5, max:10});
  },
  valid() {
    return true;
  },
```

In that case it would be possible to generate `coupons` easily in tests:
```js
const coupon = server.create('coupon');
const coupons = server.createList('coupon', 100);
```

Mirage is very powerful, you can find more in its docs: https://www.ember-cli-mirage.com/docs/getting-started/what-is-mirage

Reply:
What was wrong with faker? Why didn't it work?